src/app/shipping/ShippingHeader.tsx

```diff
@@ -32,7 +32,7 @@ const ShippingHeader: FunctionComponent<ShippingHeaderProps> = ({
             <a
                 data-test="shipping-mode-toggle"
                 href="#"
-                onClick={ preventDefault(onMultiShippingChange) }
+                onClick={ preventDefault(() => onMultiShippingChange(isMultiShippingMode)) }
```

Comment:
Do you need this change? AFAIK `isMultiShippingMode` is passed down from `Shipping`, so there's no need to pass it back up.